changedetectionio/tests/test_notification.py

```diff
@@ -142,6 +153,7 @@ def test_check_notification(client, live_server):
     assert "preview/" in notification_submission
     assert ":-)" in notification_submission
     assert "New ChangeDetection.io Notification - {}".format(test_url) in notification_submission
+    assert testimage in notification_submission
```

Comment:
```suggestion
    assert base64.b64decode(testimage) in notification_submission
```
right?

Reply:
no, as it is base64 encoded in the json output of apprise.